### PR TITLE
neofs-adm: include neofs contracts in a custom group

### PR DESCRIPTION
**Pull request by fyrchik**
Related #749 .

Morph client will be changed in another PR.
Testing is done on this branch https://github.com/fyrchik/neofs-adm-tests/tree/force-new-epoch

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/epoch.go [hunk 1]
```diff
@@ -30,10 +31,21 @@ func forceNewEpochCmd(cmd *cobra.Command, args []string) error {
...
+	groupKey, err := nnsResolveKey(wCtx.Client, cs.Hash, groupKeyDomain)
+	if err != nil {
+		cmd.Println("Can't resolve neofs contract group key, using Global scope")
+	} else {
+		signer.Scopes = transaction.CustomGroups
+		signer.AllowedGroups = keys.PublicKeys{groupKey}
+	}
+
+	res, err := wCtx.Client.InvokeFunction(nmHash, "epoch", []smartcontract.Parameter{},
+		[]transaction.Signer{signer})
```
Comment (alexvanin) [inline cmd/neofs-adm/internal/modules/morph/epoch.go:48]: Is it required to specify scope for read-only operations like fetching epoch number? I would rather keep them as simple as possible, what do you think?

File: cmd/neofs-adm/internal/modules/morph/initialize.go [hunk 2]
```diff
@@ -28,6 +28,8 @@ type initializeContext struct {
 	// ConsensusAcc is used for retrieving committee address and verification script.
 	ConsensusAcc *wallet.Account
 	Wallets      []*wallet.Wallet
+	// ContractWallet is a wallet for providing contract group signature.
+	ContractWallet *wallet.Wallet
```
Comment (alexvanin) [inline cmd/neofs-adm/internal/modules/morph/initialize.go:32]: It called `manifest wallet` in code above.

File: cmd/neofs-adm/internal/modules/morph/generate.go [hunk 3]
```diff
@@ -42,6 +42,11 @@ func generateAlphabetCreds(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	err = initializeManifestWallet(walletDir)
+	if err != nil {
+		return err
+	}
```
Comment (alexvanin) [inline cmd/neofs-adm/internal/modules/morph/generate.go:48]: Correct me if I am wrong: in the case when we already have all alphabet wallets, we call `generate-alphabet` to generate new wallets but we will take only new manifest wallet from that.

**Review by alexvanin — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/morph/epoch.go (hunk 1 above)
Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/epoch.go:48]: Fixed.

File: cmd/neofs-adm/internal/modules/morph/generate.go (hunk 3 above)
Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/generate.go:48]: I think we can initialize it during deploy if it is missing.

File: cmd/neofs-adm/internal/modules/morph/initialize.go (hunk 2 above)
Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/initialize.go:32]: Fixed.

File: cmd/neofs-adm/internal/modules/morph/generate.go (hunk 3 above)
Comment (alexvanin) [reply cmd/neofs-adm/internal/modules/morph/generate.go:48]: So `generate-alphabet` will not rewrite existing wallets now, right? Or it needs to be implemented separately?

File: cmd/neofs-adm/internal/modules/morph/epoch.go (hunk 1 above)
Comment (alexvanin) [reply cmd/neofs-adm/internal/modules/morph/epoch.go]: I still see this snippet below, maybe there is some misunderstanding. Is it necessary to specify group in read-only method invocations? I don't mind to have it, but it is a bit useless in this case, IMO.
```go
	} else {
		signer.Scopes = transaction.CustomGroups
		signer.AllowedGroups = keys.PublicKeys{groupKey}
	}
```

Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/epoch.go]: No, it is not necessary.

File: cmd/neofs-adm/internal/modules/morph/generate.go (hunk 3 above)
Comment (fyrchik) [reply cmd/neofs-adm/internal/modules/morph/generate.go:48]: Current behaviour:
1. `generate-alphabet` creates alphabet wallets and a contract wallet, overwrites all existing files.
2. `morph init` opens all wallets from disk and creates a contract wallet if it is missing.

**Pull request merged**